### PR TITLE
Don't `.show()` the main window if Kap was opened on system startup

### LINKED ISSUE #208: Don't open on system startup
The app should open, but not the main window 😊

### PR DESCRIPTION
**Pull request by matheuss**
Fixes #208 👌 

Also bumped the `ffmpeg` version since the current one is gone

### CODE REVIEW

File: app/src/main/index.js [hunk 1]
```diff
@@ -39,6 +39,10 @@ let recording = false;
 
 settings.init();
 
+const wasOpenedAtLogin = app => {
+  return app.getLoginItemSettings().wasOpenedAtLogin;
+};
```
Comment (sindresorhus) [inline app/src/main/index.js]:
```js
const wasOpenedAtLogin = app => app.getLoginItemSettings().wasOpenedAtLogin;
```

File: app/src/main/index.js [hunk 2]
```diff
@@ -294,7 +298,12 @@ menubar.on('after-create-window', () => {
       // The `move` event is called when the user reselases the mouse button
       // because of that, we need to move the window to it's expected position, since the
       // user will never release the mouse in the *right* position (diff.[x, y] === 0)
-      tray.setHighlightMode('always');
+      if (!wasOpenedAtLogin(app)) {
+        // Seems like electorn fires the `move` event right aftet the app boots,
```
Comment (sindresorhus) [inline app/src/main/index.js]: electorn => Electron

aftet => after

compiuter => computer

Comment (matheuss) [inline app/src/main/index.js]: Seems like my PhD in typos is still valid 😂

File: app/src/main/index.js [hunk 3]
```diff
@@ -294,7 +296,12 @@ menubar.on('after-create-window', () => {
       // The `move` event is called when the user reselases the mouse button
       // because of that, we need to move the window to it's expected position, since the
       // user will never release the mouse in the *right* position (diff.[x, y] === 0)
-      tray.setHighlightMode('always');
+      if (!wasOpenedAtLogin(app)) {
```
Comment (matheuss) [inline app/src/main/index.js]: This introduces a bug – tray will never be highlighted if this check is in place and the app was opened on system startup 🐛

**Pull request closed without merging**